### ISSUE TITLE
Fix the Moneydance column dance

### ISSUE THREAD

**Issue by lordpixel23** [TheInfiniteKind/moneydance_open#121]
<img width="788" alt="image" src="https://github.com/user-attachments/assets/ce8b8434-1e53-4fdd-ac3e-77f3e75cdca0">

Here is a transaction with an incorrect category I want to edit. Note how the Confirm/Merge panel on the right hand side is *open*.

1. Double Click on the Category drop down to edit
2. The Confirm/Merge panel disappears and everything jumps to the right
3. Because this causes the column to shift, I am now editing the Description field instead of the field I clicked on
4. Tab/click again to reselect the Category column and find the right category
5. Hit Enter/Return a few times to get it to save
6. The Confirm/Merge panel re-appears and everything jumps to the left

Of course, it depends exactly where I click in the Category column whether this reproduces or not, because it is about the table moving under the mouse pointer. If I am far enough over my click still puts focus in the Category field. If not, I end up editing Description.

This is probably the single thing which bugs me the most about using Moneydance day to day. To be forever clicking and ending up editing not the thing I clicked on but the wrong column.

**Comment by yogi1967** [comment]
Andrew. FYI - This is the wrong place for IK support. You should repost on the proper forum:

https://infinitekind.tenderapp.com/discussions

**Comment by lordpixel23** [comment]
OK. I'll refrain from adding any more tickets here. Your house, your rules. 

I did it for a couple of reasons
1. someone from IK told me it was OK on Twitter
![IMG_277D2FAC8395-1](https://github.com/user-attachments/assets/c7a6fb1d-3cc0-4538-81b4-0313b5a87db3)

2. It works—you yourself updated one of my tickets when an issue was fixed, for which I am still very grateful.

https://github.com/TheInfiniteKind/moneydance_open/issues/84#issuecomment-1987336783

To be honest my perception of the support forums is that while folks try to help with issues like "can't connect to bank X", anything resembling a bug report or a feature request will be ignored. And anything posted there expires after a while and there's no workflow. I can't see if anyone worked on it. I can't track closure.

A support forum is a support forum and a bug tracker is a bug tracker. If this isn't the right bug tracker, maybe IK should make one which is? Yes, you'd get spam there. That can't be helped. Everything is a tradeoff.

**Comment by yogi1967** [comment]
(It's not my house ;-> ). Nothing to do with me. I'm guiding you that the official support forum is elsewhere. Yes, there are a couple of people that watch this and if you're lucky you might get a reply. You will almost always get a reply on the forum.. Just IMHO. 

This hub is for open source extensions and a few experts to converse.... but there's nothing stopping you posting here.

**Comment by lordpixel23** [comment]
Yeah. I will give it a go. Would you prefer I close all these tickets?
